tests/Keycloak/Listeners/CreateMissingClientsHandlerTest.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Keycloak\Listeners;
+
+use App\Domain\Integrations\Integration;
+use App\Domain\Integrations\Repositories\IntegrationRepository;
+use App\Keycloak\Client;
+use App\Keycloak\ClientCollection;
+use App\Keycloak\Jobs\MissingClientsDetected;
+use App\Keycloak\Listeners\CreateMissingClients;
+use App\Keycloak\Realm;
+use App\Keycloak\RealmCollection;
+use App\Keycloak\Repositories\KeycloakClientRepository;
+use App\Keycloak\ScopeConfig;
+use App\Keycloak\Service\ApiClient;
+use App\Keycloak\Service\CreateClientForRealms;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use Psr\Log\LoggerInterface;
+use Ramsey\Uuid\Uuid;
+use Tests\IntegrationHelper;
+
+final class CreateMissingClientsHandlerTest extends TestCase
```

Comment:
The name should be `CreateMissingClientsTest`

Reply:
Test is removed because merged with CreateClientsTest